ui/app/AppLayouts/Profile/views/SyncingView.qml

```diff
@@ -272,6 +271,30 @@ SettingsContentBase {
                     }
                 }
 
+                RowLayout {
+                    Layout.fillWidth: true
+                    Layout.alignment: Qt.AlignHCenter
+                    spacing: Theme.halfPadding
+
+                    StatusCheckBox {
```

Comment:
It's not overflow-safe (narrow popup, long translated text). To fix it we could add e.g.:
```
Layout.fillWidth: true
Layout.maximumWidth: implicitWidth
```